plugins/modules/identity/keycloak/keycloak_realm.py

```diff
@@ -657,7 +658,7 @@ def main():
         reset_password_allowed=dict(type='bool', aliases=['resetPasswordAllowed']),
         revoke_refresh_token=dict(type='bool', aliases=['revokeRefreshToken']),
         smtp_server=dict(type='dict', aliases=['smtpServer']),
-        ssl_required=dict(type='bool', aliases=['sslRequired']),
+        ssl_required=dict(choices=["external", "all", "none"], aliases=['sslRequired']),
```

Comment:
Is using `none` by default appropriate in `keycloak`? Does this have any effect on current users using this module? If answer is yes to first question and no to second question, please also add default to documentation ;)
```suggestion
        ssl_required=dict(choices=["external", "all", "none"], aliases=['sslRequired'], default="none"),
```

Reply:
Hi @aminvakil ,
For me put a default value is not a good practice because keycloak puts his own default values when you don't specify it

Reply:
@kris2kris Right, this will also create problem when keycloak changes its default.
@BenjaminSchubert Feel free to ignore my suggestion.

Reply:
> @kris2kris Right, this will also create problem when keycloak changes its default.

I'd argue that people that want to avoid surprises could also set the default themselves :)

Also 'None' as the default is the most unsafe of those choices, so yeah I'd rather let Keycloak choose too.

As such I'll resolve this comment